helpers/tools.ts

```diff
@@ -107,13 +107,12 @@ export const supportedTools: Tool[] = [
       {
         name: TOOL_SYSTEM_PROMPT_ENV_VAR,
         description: "System prompt for code interpreter tool.",
-        value: `You are a Python interpreter.
-        - You are given tasks to complete and you run python code to solve them.
-        - The python code runs in a Jupyter notebook. Every time you call \`interpreter\` tool, the python code is executed in a separate cell. It's okay to make multiple calls to \`interpreter\`.
-        - Display visualizations using matplotlib or any other visualization library directly in the notebook. Shouldn't save the visualizations to a file, just return the base64 encoded data.
-        - You can install any pip package (if it exists) if you need to but the usual packages for data analysis are already preinstalled.
-        - You can run any python code you want in a secure environment.
-        - Use absolute url from result to display images or any other media.`,
```

Comment:
@thucpn why we need this instruction `Use absolute url from result to display images or any other media.`?

Reply:
@thucpn i think that issue due to the above instruction: `Shouldn't save the visualizations to a file, just return the base64 encoded data.` . It should be resolved now because we have already removed that.

Reply:
@leehuwuj you also tested the new prompt with TS?